frameworks/g-cloud-9/questions/services/approachToResilience.yml

```diff
@@ -4,6 +4,11 @@ question_advice: >
   Include how your datacentre setup is resilient. If you don’t want to make this information public, you can say that
   it’s available on request.
 
+  Read about the government’s <a
+  href="https://www.ncsc.gov.uk/guidance/cloud-security-principle-2-asset-protection-and-resilience#resilience"
+  target="_blank" rel="noopener noreferrer">2nd cloud security principle: ‘Asset protection and resilience’ (link opens
+  in a new tab)</a>.
```

Comment:
The 'link opens in new tab' bit is best practice for accessibility according to some docs given me by @laurenceberry.